XCTest/LinuxMain.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
```

Comment:
I haven’t removed your test executable. You can still use it the way you have been when you test manually.

Workspace checks that `$ swift test` passes for the package. (And there is currently no way to allow test failures.) `$ swift test` fails if the package has no (conventional) tests. I considered just creating a blank test, but I thought it made more sense for it to [run your custom test executable as an external process](https://github.com/kelvin13/png/pull/12/files#diff-7c0e705781491f42db9d3b4e4c319cdcR11), so that `$ swift test` gives a meaningful result. It will throw (thus failing) if the exit code is non‐zero. 

Basically, as it is, these now do exactly the same thing:
```
$ swift build --configuration release
$ .build/release/tests

$ swift test
```

The external process is still built externally in release mode (optimized), so your performance tests aren’t affected by any debug stuff.

***

If you would instead rather just have an empty test method, you could [disable continuous integration management](https://github.com/kelvin13/png/pull/12/files#diff-04be69ad05210c4e7a0d34b80882813cR16), and manually add a separate custom job to `.travis.yml` for each OS to run the tests separately.

Eventually, when SDGGiesbrecht/workspace#195 is implemented, you will be able to include a custom executable as a validation step, but that functionality is not available yet.

Reply:
Also gone now.